x-pack/plugins/fleet/public/components/package_policy_actions_menu.tsx

```diff
@@ -117,7 +117,10 @@ export const PackagePolicyActionsMenu: React.FunctionComponent<{
               disabled={!hasWriteCapabilities}
               icon="trash"
               onClick={() => {
-                deletePackagePoliciesPrompt([packagePolicy.id], refreshAgentPolicy);
+                deletePackagePoliciesPrompt([packagePolicy.id], () => {
+                  setIsActionsMenuOpen(false);
```

Comment:
Tangential change here. Previously, when deleting an integration policy, we didn't close the action menu which led to some odd UI states. This ensures we close the menu once the delete action is completed.